src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/PlatformManager.java

```diff
@@ -126,12 +128,11 @@ public Payment getPlatformPayment(String name) {
      *
      * @param name The name of the platform
      * @param experiment The experiment to publish
-     * @return None if the platform does not exist
+     * @throws PreActionException if this gets thrown the experiment is not published.
+     * @return A completable future object which indicates if the publishing was successful or not
```

Comment:
i would still improve that a little bit...like: a normally completed future represents x, an exceptionally completed future represents failure